tasks/fetch_ward_data.py

```diff
@@ -52,10 +53,28 @@ def run(self):
         repository.store_dataframe(all_wards, ward_storage_location, allow_create=True, store_index=True)
 
         # impute delta's atleast for Mumbai this is needed it only provides totals
+        delta_needed_for = ['total.confirmed', 'total.recovered', 'total.deceased', 'total.active']
+        group_by_cols = ['state', 'district', 'ward']
+        all_wards = create_delta_cols(all_wards,group_by_cols, delta_needed_for)
+
         all_wards.to_csv(self.output().path, index=True)
```

Comment:
drop the .total. in delta.total.metric_name
THis to keep it in sync with states and district naming convention